app/components/widgets/forms/social-link-field.hbs

```diff
@@ -1,4 +1,5 @@
-<div class="fields">
+<div class="{{if @addLinks 'fields'}}">
+  {{#if @addLinks}}
```

Comment:
Why addLinks?

Reply:
ofcourse we could give this more sensible name but this is bcz at some places we dont need that UiDropdrop and input defined in social-link-field

Reply:
If no dropdown is needed, why not use LinkField instead of SocialLinkField?

Reply:
then I have to manually provide prefix value for each of them for e.g
```@prefix={{'https://twitter.com/'}}```

Reply:
If this is ok then I can do like this

Reply:
Just change prop to `fixed` and if fixed is true, dropdown will not be shown

Reply:
rename addLinks to fixed?

Reply:
Invert it